### PR TITLE
Implementing Source DBMS DB2

### PR DESCRIPTION
**Pull request by thalesph**
Hi Folks,

By a bad decision I installed OTRS 3.1 with DB2 as DBMS. Since then, about 2013, and forced with the new releases, we prioritize to migrate to MySQL.

After some interaction from our DB2 DBA, we had some problems when migrating BLOBs. 

So, I extend CloneDB Plugin and to support DB2 DBMS as SOURCE, after some minor fixes, today we finished our migration from DB2 to MySQL with success.

TL;DR: I extend CloneDB plugin to clone DB2 to MySQL.

### CODE REVIEW

**Comment by mgruner** [comment]
Hello @thalesph. Thanks a lot for your contribution, however it seems that the changes in this PR were made for an older version of OTRS and this module. OTRS 5 now uses the object manager in its code, and also does not have a DB2 driver in its codebase, so that this one would have to be added as well.
Please feel free to reopen this once it is updated to the master version of this package. 
@carlosgarciac: FYI.
Best regards, Martin
